frontend/sw360-portlet/src/main/webapp/component/searchfield.jspf

```diff
@@ -1,4 +1,16 @@
-<div id="component-searchField" class="content1">
+<%--
+ ~ Copyright COPYRIGHT HOLDER, 2017.
```

Comment:
I believe you're supposed to replace "COPYRIGHT HOLDER" with the actual name of the copyright holder :-)